pkg/grafeas/helper.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 The Rode Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grafeas
+
+import (
+	"context"
+	"fmt"
+	"github.com/rode/rode/pkg/constants"
+	grafeas_proto "github.com/rode/rode/protodeps/grafeas/proto/v1beta1/grafeas_go_proto"
+	"go.uber.org/zap"
+	"strings"
+)
+
+//go:generate counterfeiter -generate
+
+//counterfeiter:generate . Helper
+type Helper interface {
```

Comment:
I'm open to feedback on a better name for this

Reply:
`GrafeasExtensions`?

Reply:
so maybe we should come to a consensus on this too... should the type itself be called `GrafeasExtensions`, meaning it would be referenced as `grafeas.GrafeasExtensions` in other packages?  or do we call the type `Extensions` / `grafeas.Extensions`?  Goland yells at me if the name of the type starts with the name of the package 😄

Reply:
Seems like the [Go convention](https://github.com/golang/go/wiki/CodeReviewComments#package-names) would be to call it `Extensions` so that it could be used as `grafeas.Extensions`. I like that the succinctness of that, although it does cause problems with embedding. I think I'd vote to avoid the package name in the type unless it would cause a collision.